### ISSUE TITLE
Fork to Japanese translation mod

### ISSUE THREAD

**Issue by monakaJP** [Eltirosto/Degrees-of-Lewdity-Chinese-Localization#413]
### 在提问之前...

- [X] 我填写了简短且清晰明确的标题，以便开发者在翻阅 issue 列表时能快速确定大致问题。而不是“一个建议”、“卡住了”等
- [X] 我确定这既不是对游戏报错的反馈，也不是对汉化文本改进的建议

### 说说你遇到的问题？

Hello, I am a Japanese developer trying to translate dol into Japanese. At first . I was developing it by translating the original source code as it is, but I was stuck with a lot of bugs. Then I found this translation mod. May I fork this project and create a translation dictionary for Japanese?

**Comment by NumberSir** [comment]
Yes 😁 but what you want to fork might be [this repo](https://github.com/NumberSir/DoL-I18n-Build), which is our i18n mod auto-built repo, not current repo. 

You can check [this repo](https://github.com/Lyoko-Jeremie/sugarcube-2-ModLoader) and [this repo](https://github.com/Lyoko-Jeremie/Degrees-of-Lewdity_Mod_i18nMod) for more info.

**Comment by monakaJP** [comment]
Thank you! The tools have already been developed and are very helpful! I will fork these repo.

**Comment by monakaJP** [comment]
Thank you! The tools have already been developed and are very helpful! I will fork these repo.
